content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Example of MapReduce With Ezmeral Data Fabric(MapR) Database Binary Table
+date: 2022-11-14T15:56:03.596Z
+featuredBlog: false
+author: Raymond Yan
+authorimage: /img/raymondyan-profile.png
+thumbnailimage: /img/mapreduce_with_edf_binarytable.png
+disable: false
+tags:
+  - hpe-ezmeral-data-fabric
+  - MapReduce
+---
+
+## Introduction
+
+
+
+This article will cover:
+
+
+
+1. How to create Development Environment for HPE Ezmeral Data Fabric on Linux, Windows and Mac.
+
+
+
+    This is a one-node cluster based on Docker containers, with a choice of different EDF versions, it integrates EEP.
+    This way you can quickly create an EDF environment on your work computer.
+
+
+
+2. Demonstrate a MapReduce application that uses EDF's Database Binary Table as the backend service.
+
+
+
+    I will create the table using the hbase shell command line tool customized for EDF Database and do CRUD operations using a MapReduce application.
```

Comment:
...and do CRUD (Create, Read, Update, Delete) operations using a MapReduce application.